email.sh

```diff
@@ -52,6 +43,13 @@ else
   EVERGREEN=evergreen
 fi
 
+# Fetch previous release tag and get the commits since that tag
+CURRENT_COMMIT_HASH=$(git rev-parse --short HEAD)
+PREVIOUS_TAG=$(git describe --abbrev=0 $CURRENT_COMMIT_HASH\^)
+# get all commits since the previous tag
+git log --no-merges $PREVIOUS_TAG..$CURRENT_COMMIT_HASH --pretty="%h %s" >> body.txt
```

Comment:
Should we really include these commits in the case of a revert?

E.g., Imagine we deploy commit C, then a few days later B, then a few days later A. We then discover we need to revert A. When we deploy B, we include the commits between C and B. Would that be confusing? That suggests these are _newly_ deployed commits, when in fact they've been in prod the whole time.

Reply:
Good point i updated the revert email to only show the currently deployed hash.
It uses the presence of a non 0 execution to trigger the revert case. 
![image](https://user-images.githubusercontent.com/4605522/171673731-4cb386a8-8e0f-4fb7-a9d0-9d30f2df9b29.png)